packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1187,7 +1188,7 @@ export class Combobox
 
   renderInput(): VNode {
     const { guid, disabled, placeholder, selectionMode, selectedItems, open } = this;
-    const single = selectionMode === "single";
+    const single = selectionMode === "single" || selectionMode === "single-persist";
```

Comment:
could this be `const single = isSingleLike(selectionMode)`?

Reply:
Nice catches! 🎣